.github/workflows/update-flake.yml

```diff
@@ -0,0 +1,23 @@
+name: Update flake inputs
+on:
+  schedule:
+    # Update every Sunday and Wednesday
+    - cron: "51 3 * * 0,3"
```

Comment:
I still think twice a week is excessive and will lead to PRs piling up. How about once a week?

Reply:
It wouldn't really pile up, right? It would just update an already open PR. 

It seems more important to have frequent updates available, even if the PR is sometimes left open for a while.

Reply:
Ah, it seems you're right. In that case it's probably fine.